src/classes/GE_LookupController.cls

```diff
@@ -47,8 +47,49 @@ public with sharing class GE_LookupController {
      *
      * @return              A List<LookupResult> containing records that match the search criteria.
      */
-    @AuraEnabled(cacheable=true)
+    @AuraEnabled(Cacheable=true)
```

Comment:
I know this is persisting from before, but is the use of "cachable=true" going to mess with any attempts to add new records on the fly or anything like that?

Reply:
It could, here's what would have to happen:

1. Search for "Astro", but get 0 results because no contact exists yet.
2. Create "Astro Donor" as a contact and save.
3. Search for "Astro" again, 0 results returned because the exact same search string was used.

If the user searches for "Astro D" or "Donor" they should see the correct results but if the same search is done, the same results will come back.

Reply:
I can definitely think of some weirdness coming about as a result of applying caching here - I actually do open a new browser tab and add a new contact, campaign, or GAU relatively frequently while I'm using Gift Entry, although I'm probably not super typical. Do we have a sense for what the performance impact is if we take away this caching?